src/System.Management.Automation/engine/debugger/debugger.cs

```diff
@@ -971,7 +971,7 @@ internal ScriptDebugger(ExecutionContext context)
             _context = context;
             _inBreakpoint = false;
             _idToBreakpoint = new ConcurrentDictionary<int, Breakpoint>();
-            _pendingBreakpoints = new ConcurrentDictionary<int, LineBreakpoint>();
+            _pendingBreakpoints = new ConcurrentDictionary<string, ConcurrentDictionary<int, LineBreakpoint>>();
```

Comment:
The string key is function context file path. The int key is sequencePoint index.

Reply:
Best to add that as a comment here